src/mlpack/methods/pca/decomposition_policies/quic_svd_method.hpp

```diff
@@ -50,8 +50,8 @@ class QUICSVDPolicy
    * @param eigvec Matrix to put eigenvectors (loadings) into.
    * @param rank Rank of the decomposition.
    */
-  void Apply(const arma::mat& data,
-             const arma::mat& centeredData,
+  void Apply(arma::mat& data,
+             arma::mat& centeredData,
```

Comment:
Once we change the `CosineTree` to take a reference again, this change should not be needed.